packages/ripple/package.json

```diff
@@ -22,9 +22,6 @@
   "dependencies": {
     "@leafygreen-ui/tokens": "^1.3.4"
   },
-  "devDependencies": {
-    "@leafygreen-ui/lib": "^10.0.0"
```

Comment:
does ripple need react? if so this won't work

Reply:
It's only actually needed as a devDependency (in spec files)